internal/service/service.go

```diff
@@ -1 +1,41 @@
 package service
+
+import (
+	"errors"
+	"log"
+	"os"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/pkg/morse"
+)
+
+func convertMorse(text string) string{
+	if isMorse(text){
+		return morse.ToText(text)
+	}
+	return morse.ToMorse(text)
+}
+
+func isMorse(text string) bool{  // 46 = "."; 45 = "-"; 32 = " "
+	for _, letter := range(text) {
+		if letter != 46 && letter != 45 && letter != 32 {
```

Comment:
не используй "магические цифры"
```suggestion
		if letter != '.' && letter != '-' && letter != ' ' {
```